packages/kbn-test/src/es/es_test_cluster.js

```diff
@@ -78,7 +79,9 @@ export function createEsTestCluster(options = {}) {
         esArgs: [
           `cluster.name=${clusterName}`,
           `http.port=${port}`,
+          `node.name=${nodeName}`,
           `discovery.zen.ping.unicast.hosts=localhost:${port}`,
```

Comment:
I think this should be empty:

```suggestion
          `discovery.zen.ping.unicast.hosts=[]`,
```

There's no need to mention the local node in this list, and anyway it's referring to the HTTP port not the transport port.